src/Compilers/VisualBasic/Portable/Binding/Binder_Expressions.vb

```diff
@@ -800,7 +800,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             For Each diag As Diagnostic In tree.GetDiagnostics(root)
                 Dim cdiag = TryCast(diag, DiagnosticWithInfo)
                 Debug.Assert(cdiag Is Nothing OrElse Not cdiag.HasLazyInfo,
-                             "If we decide to allow lazy syntax diagnostics, we'll have to check all call sites of SyntaxTree.GetDiagnostics")
+                             "If we decide To allow lazy syntax diagnostics, we'll have to check all call sites of SyntaxTree.GetDiagnostics")
                 If diag.Severity = DiagnosticSeverity.Error Then
```

Comment:
undo this change. Looks like the pretty lister tried to fix this maybe when you opened a string literal or something.